### DISCUSSION TITLE
How did you integrate the js scripts into the html page?

### DISCUSSION THREAD

**Discussion by 45675** [nroutasuo/level13#213] [Q&A] [answered]
Hey, I am getting into HTML and JS for making a game and I was wondering how you integrated the js scripts into the html without using <script> in the html. How would this be done, and how did you do it?

**Comment by Toby222** [comment]
There's definitely <script> tags in the html
![image](https://user-images.githubusercontent.com/14962962/139872269-aaa26981-60a8-4795-be7e-b8f809bdaaa8.png)

**Comment by nroutasuo** [comment, marked as answer]
You will see a few script tags at the bottom of the original [.html file](https://github.com/nroutasuo/level13/blob/master/index.html), but most of the tags in the @Toby222 's screenshot are generated. I use a JavaScript module loader called [RequireJS](https://requirejs.org/). This is what refers to it in the html file:
`	<script data-main="src/config" src="lib/requirejs/require.min.js"></script>`

This project was started a long time ago and JavaScript evolves quite fast, so I'm not sure this approach is the way to go today. I think in modern JavaScript it's possible to do something similar without a 3rd party library like Require. But it works well for Level 13.

**Reply by 45675** [reply]
that makes a lot of sense.
